resources/js/components/VTRomSettings.vue

```diff
@@ -148,6 +159,13 @@ export default {
     };
   },
   methods: {
+    loadCustomSprite(e) {
+      if (e) {
+        this.showLoadCustomSprite = true;
+      } else {
+        this.showLoadCustomSprite = false;
+      }
```

Comment:
Could this be?
```js
this.showLoadCustomSprite = Boolean(e);
```

Reply:
Good idea; done.

Regarding the other suggested TODOs, I was hoping this PR could be merged, then others could assist in the enhancements; some of them are beyond my current ability with vue/bootstrap/JS/etc..  Are there any in particular you'd insist on being completed before merging this PR?